.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Guidelines to submit an issue
+
+If you are opening an issue because you woul like to propose a new feature,
+please write the title as "Feature Request:" plus a small description of your
+suggestion.
+
+If you are opneing an issue because you found a bug, please make sure that
+your have the latest version installed. If yes then describe it by following
+these guidelines:
+
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce the problem
+
+
+### Your operational system
```

Comment:
Ok. I preferred the long version and added some examples so it will be no mistake.